packages/api/src/controllers/session.ts

```diff
@@ -145,13 +145,24 @@ app.get("/:id", authorizer({}), async (req, res) => {
   let session = await db.session.get(req.params.id);
   if (
     !session ||
-    ((session.userId !== req.user.id || session.deleted) && !req.user.admin)
+    ((session.userId !== req.user.id || session.deleted) &&
+      !req.user.admin &&
+      !LVPR_SDK_EMAILS.includes(req.user.email))
   ) {
     // do not reveal that session exists
     res.status(404);
     return res.json({ errors: ["not found"] });
   }
   res.status(200);
+
+  let originalRecordingUrl: string | null = null;
+
+  if (req.query.sourceRecording == "true") {
```

Comment:
use triple equals in javascript.
```suggestion
  if (req.query.sourceRecording === "true") {
```